packages/blade-coverage-plugin/README.md

```diff
@@ -0,0 +1,33 @@
+# Blade Coverage Plugin
```

Comment:
Are we planning to publish this plugin somewhere for easier access?

Reply:
Yes, with a one time fees it will be published to chrome extensions and everyone can use it.